sim/druid/feral/TestFeral.results

```diff
@@ -2022,6 +2022,14 @@ dps_results: {
   hps: 8558.02845
  }
 }
+dps_results: {
+ key: "TestFeral-AllItems-RuneofRe-Origination-96918"
+ value: {
+  dps: 109953.34538
+  tps: 188248.14133
+  hps: 8497.04677
+ }
+}
```

Comment:
Make sure to update the test file for this result, since the true value is now higher with the snapshotting callback included.